scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/Schemas.java

```diff
@@ -105,4 +105,7 @@ public static Schemas getSchemaByNamespace(String namespace) {
         return SCHEMAS_BY_NAMESPACE.get(namespace);
     }
 
+    public String getVersion() {
+        return namespace.substring(namespace.lastIndexOf(":") + 1);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Put single-quotes around ':' to use the faster "lastIndexOf(char)" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3027)